static_src/components/activity_log.jsx

```diff
@@ -74,7 +76,9 @@ export default class ActivityLog extends React.Component {
   render() {
     let content = <div></div>;
 
-    if (this.state.empty) {
+    if (this.state.empty && this.state.hasErrors) {
+      content = <h5 className="test-none_message">An error occured fetching recent activity</h5>;
```

Comment:
In the current case of the activity log, an error does occur, but some activity items are still fetched (at least thats what appears to happen in my UI). Would it be better to display any items that we have and the error, or should we just display the error?

I'm worried the UI will actually get worse if we just show the error, because this error seems to happen a lot on the app page (we might want to figure out why), so we could be making the activity log unusable.